packages/react/src/components/DashboardEditor/DashboardEditorCardRenderer.test.jsx

```diff
@@ -230,4 +230,71 @@ describe('DashboardEditorCardRenderer', () => {
     );
     expect(screen.getByText(/defaultCard/)).toBeInTheDocument();
   });
+
+  it('should call getValidDataItems when passed', () => {
+    const getValidDataItems = jest.fn();
+    render(
+      <DashboardEditorCardRenderer
+        {...commonProps}
+        size={CARD_SIZES.MEDIUM}
+        getValidDataItems={getValidDataItems}
+      />
+    );
+
+    expect(getValidDataItems).toHaveBeenCalledWith({ size: CARD_SIZES.MEDIUM });
+  });
+
+  // it('should call onCardChange when image card is updated', () => {
```

Comment:
Should we uncomment or remove?

Reply:
nice catch. I started to test it and then found out it can't be tested completely in isolation in the renderer and forgot to remove it. I'll test it in another location with an ImageCard later.